cypress/plugins/index.js

```diff
@@ -47,5 +47,18 @@ module.exports = (on, config) => {
     }
   });
 
+  on('task', {
```

Comment:
Is this required? Does it have a different effect from cy.log()? Or is this a requirement because of cypress-axe? If the latter, might be worth commenting here that this is only in place to support cypress-axe (in case we ever changed this dependency or removed it).

Reply:
OK comment added